src/modules/showReservation.js

```diff
@@ -0,0 +1,62 @@
+const showReservation = (obj) => {
+  const {
+    short_title: shortTitle,
+    performers: [{ image: performerImage }],
+    type: tipo,
+    datetime_local: dateEvent,
+    venue: { city: ciudad, country: pais, state: estado },
+    stats: { average_price: precio },
```

Comment:
Kindly use English variable names in the codebase as it will make it more readable and understandable for other developers who might be working on the project, regardless of their language background.